packages/e2e-tests/specs/navigable-toolbar.test.js

```diff
@@ -29,20 +22,19 @@ describe( 'block toolbar', () => {
 			!! document.activeElement.closest( '.block-editor-block-toolbar' )
 		) );
 
-		describe( label, () => {
-			it( 'navigates in and out of toolbar by keyboard (Alt+F10, Escape)', async () => {
-				// Assumes new post focus starts in title. Create first new
-				// block by ArrowDown.
-				await page.keyboard.press( 'ArrowDown' );
+		it( 'navigates in and out of toolbar by keyboard (Alt+F10, Escape)', async () => {
+			// Assumes new post focus starts in title. Create first new
+			// block by ArrowDown.
+			await page.keyboard.press( 'ArrowDown' );
 
-				// [TEMPORARY]: A new paragraph is not technically a block yet
-				// until starting to type within it.
-				await page.keyboard.type( 'Example' );
+			// [TEMPORARY]: A new paragraph is not technically a block yet
+			// until starting to type within it.
+			await page.keyboard.type( 'Example' );
 
-				// Upward
-				await pressKeyWithModifier( 'alt', 'F10' );
-				expect( await isInBlockToolbar() ).toBe( true );
-			} );
+			// Upward
+			await pressKeyWithModifier( 'alt', 'F10' );
+			expect( await isInBlockToolbar() ).toBe( true );
```

Comment:
This is apparently failing now. Will need to investigate which change exactly caused this.

Reply:
It seems to be a valid failure. Focus gets moved to the inserter (first item in the unified toolbar).

I'm wondering whether it was working properly before.

Reply:
bb31814 fixed it 😄